Display/features/environment.py

```diff
@@ -0,0 +1,8 @@
+
+
```

Comment:
This is where you can do 'before_all' and start your docker container, to do testing external to the code